### PR TITLE
move HYGROMETER_SENSOR from development.xml to common.xml

### PR DESCRIPTION
**Pull request by gitfishup**
Based on the new message, I implemented the display of the hygrometer information on the ground station.
NOTE:
      I move HYGROMETER_SENSOR from development.xml to common.xml to test. Because Mission Planner and ardupilot don't have the development.xml file. So , ardupilot , MissionPlanner , PX4-Autopilot , Qgroundcontroll are use the  HYGROMETER_SENSOR message from common.xml.
      
Here，i will show you the display on QGC and MissionPlanner：
bd311c6af9f0bf0fe303c4e090c7b6a
1d86cf6814e4c35d9c48e91b3a53414
And here is the videos  show that the dynamic change of hygrometer message(The airspeed is heating)

https://user-images.githubusercontent.com/69180601/129299677-2b9b0533-860e-42f5-9239-96bafb46f90f.mp4

https://user-images.githubusercontent.com/69180601/129299686-8b70d95a-ddbf-4be8-ac32-35a1f6b4c08d.mp4

@hamishwillee if you need more details about test , i wil show you .

### CODE REVIEW

**Comment by gitfishup** [comment]
@hamishwillee Could you please have review?

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -6893,5 +6893,13 @@
       <field type="uint8_t" name="msg_pack_size">Number of encoded messages in the pack (not the number of bytes). Allowed range is 1 - 10.</field>
       <field type="uint8_t[250]" name="messages">Concatenation of encoded OpenDroneID messages. Shall be filled with nulls in the unused portion of the field.</field>
     </message>
+    <message id="12920" name="HYGROMETER_SENSOR">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>temperature and humidity from hygrometer.</description>
+      <field type="uint8_t" name="id" instance="true">Hygrometer ID</field>
+      <field type="int16_t" name="temperature" units="cdegC">temperature</field>
+      <field type="uint16_t" name="humidity" units="c%">humidity</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:6902]: Please capitalise the description from `humidity` to `Humidity`

File: message_definitions/v1.0/common.xml [hunk 2]
```diff
@@ -6893,5 +6893,13 @@
       <field type="uint8_t" name="msg_pack_size">Number of encoded messages in the pack (not the number of bytes). Allowed range is 1 - 10.</field>
       <field type="uint8_t[250]" name="messages">Concatenation of encoded OpenDroneID messages. Shall be filled with nulls in the unused portion of the field.</field>
     </message>
+    <message id="12920" name="HYGROMETER_SENSOR">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change. It should NOT be used in stable production environments. -->
+      <description>temperature and humidity from hygrometer.</description>
+      <field type="uint8_t" name="id" instance="true">Hygrometer ID</field>
+      <field type="int16_t" name="temperature" units="cdegC">temperature</field>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:6901]: Please capitalise the description from `temperature` to `Temperature`

File: message_definitions/v1.0/common.xml [hunk 3]
```diff
@@ -6893,5 +6893,13 @@
       <field type="uint8_t" name="msg_pack_size">Number of encoded messages in the pack (not the number of bytes). Allowed range is 1 - 10.</field>
       <field type="uint8_t[250]" name="messages">Concatenation of encoded OpenDroneID messages. Shall be filled with nulls in the unused portion of the field.</field>
     </message>
+    <message id="12920" name="HYGROMETER_SENSOR">
+      <wip/>
```
Comment (hamishwillee) [inline message_definitions/v1.0/common.xml:6897]: Please remove this line and the next line (i.e. if you're moving it into common.xml is is no longer "WIP". You won't be able to change it easily after this. That is why pre-testing is important.

**Comment by hamishwillee** [comment]
Hi @gitfishup 

Thanks very much. I've added some minor comments and am nearly ready to merge. 

However this is failing CI tests. Are you able to run the formatter. Usually I do this on Ubuntu linux:

`./scripts/format_xml.sh ./message_definitions/v1.0/development.xml`

**Comment by gitfishup** [comment]
> Next time we go through through this process, logs are also useful. They show us things like message rates etc.

@hamishwillee Thanks! Do you mean the flight controller logs?

**Comment by gitfishup** [comment]
> Hi @gitfishup
> 
> Thanks very much. I've added some minor comments and am nearly ready to merge.
> 
> However this is failing CI tests. Are you able to run the formatter. Usually I do this on Ubuntu linux:
> 
> `./scripts/format_xml.sh ./message_definitions/v1.0/development.xml`
> 
> P.S. Next time we go through through this process, logs are also useful. They show us things like message rates etc.

@hamishwillee I have changed them. And i will upload the logs here soon.

**Comment by hamishwillee** [comment]
Thanks. Merging :-)

**Comment by gitfishup** [comment]
> Thanks. Merging :-)

I'm so sorry about typos. I didn't fix everything.

**Comment by hamishwillee** [comment]
No worries!

**Pull request merged**